src/components/interactive-image/interactive-image.tsx

```diff
@@ -0,0 +1,222 @@
+    }
+  }
+
+  private correctZoomValue() {
+    if (this.zoom < 100) {
+      console.warn("Zoom value cannot be lower than 100");
+      this.zoom = 100;
+    }
+  }
+
+  private editingOverClass() {
+    if (this.enableZoom && this.mouseOver) {
+      this.element.classList.add("zoom-over");
+    } else {
+      this.element.classList.remove("zoom-over");
+    }
+  }
+
+  private removeEvent(
+    element: HTMLElement,
+    eventToListen: string,
+    callbackFunction
+  ) {
+    element.removeEventListener(eventToListen, callbackFunction);
+  }
+
+  private zoomFeature = {
+    over: {
+      withMouse: "mousemove",
+      withTouch: "touchmove",
+      mouseBehaivor: ev => {
+        ev.preventDefault();
+        this.mouseOver = true;
+        this.zoomedPositionX = this.calculateZoomedPosition(
+          ev.offsetX,
+          ev.target.offsetWidth
+        );
+        this.zoomedPositionY = this.calculateZoomedPosition(
+          ev.offsetY,
+          ev.target.offsetHeight
+        );
+      },
+
+      touchBehaivor: ev => {
+        ev.preventDefault();
+        this.mouseOver = true;
+
+        const imgSize = {
+          height: ev.target.offsetHeight,
+          width: ev.target.offsetWidth
+        };
+
+        const touch = {
+          X: ev.changedTouches[0].clientX - ev.target.x,
+          Y:
+            ev.changedTouches[0].clientY -
+            ev.target.parentNode.getBoundingClientRect().top
+        };
+
+        if (touch.X <= 0) {
+          touch.X = 0;
+        } else if (touch.X >= imgSize.width) {
+          touch.X = imgSize.width;
+        }
+        if (touch.Y <= 0) {
+          touch.Y = 0;
+        } else if (touch.Y >= imgSize.height) {
+          touch.Y = imgSize.height;
+        }
+
+        const moveImgPostion = {
+          X: this.calculateZoomTouch(
+            this.calculateZoomedPosition(touch.X, ev.target.offsetWidth),
+            this.zoom
+          ),
+          Y: this.calculateZoomTouch(
+            this.calculateZoomedPosition(touch.Y, imgSize.height),
+            this.zoom
+          )
+        };
+        this.zoomedPositionX = moveImgPostion.X;
+        this.zoomedPositionY = moveImgPostion.Y;
+      }
+    },
+    out: {
+      withMouse: "mouseout",
+      withTouch: "touchend",
+      behaivor: () => {
+        this.mouseOver = false;
+        this.element.setAttribute("class", "hydrated");
+        this.element.style.backgroundPosition = `0 0`;
+      }
+    }
+  };
+
+  componentWillLoad() {
+    this.correctZoomValue();
+  }
+
+  componentDidLoad() {
+    this.checkZoomFeature();
+  }
+
+  componentDidUpdate() {
+    this.checkZoomFeature();
+  }
+
+  render() {
+    this.correctZoomValue();
+    return (
+      <img
+        style={
+          this.mouseOver
+            ? {
+                "object-position": this.mouseOver
+                  ? `${this.zoomedPositionX}px ${this.zoomedPositionY}px`
+                  : "0px 0px",
```

Comment:
No need for units when specifying `0`.
```suggestion
                  : "0 0",
```

Reply:
That was fixed in 4d5e034

Reply:
I deleted that. It was within double condition... 🤦‍♂️
Look at 4d5e034